src/views/LatticeView.vue

```diff
@@ -0,0 +1,40 @@
+<script setup lang="ts">
+import { computed, ref } from "vue";
+import type { Scale } from "scale-workshop-core";
+import { mmod } from "xen-dev-utils";
+import ScaleLattice from "@/components/ScaleLattice.vue";
+
+const props = defineProps<{
+  scale: Scale;
+  baseMidiNote: number;
+  keyColors: string[];
+  heldNotes: Map<number, number>;
+}>();
+
+const heldScaleDegrees = computed(() => {
+  const result: Set<number> = new Set();
+  for (const tableIndex of props.heldNotes.keys()) {
+    if (props.heldNotes.get(tableIndex)! > 0) {
+      result.add(mmod(props.baseMidiNote + tableIndex, props.scale.size));
```

Comment:
This should be `tableIndex - props.baseMidiNote`. Sorry it took so long to figure out.